cita-executor/core/src/contracts/native/factory.rs

```diff
@@ -46,11 +30,15 @@ impl Clone for Box<Contract> {
 
 // Contract
 pub trait Contract: Sync + Send + ContractClone {
-    fn exec(&mut self, params: &ActionParams, ext: &mut Ext) -> Result<GasLeft, evm::Error>;
+    fn exec(
+        &mut self,
+        params: &ActionParams,
```

Comment:
The struct `ActionParams` is GPL code, I will define a new one and please reference it.

Reply:
You defined one, i modified as you said once.